### DISCUSSION TITLE
Hide the new "tags you're following" thing on the dash

### DISCUSSION THREAD

**Discussion by arcanetrivia** [AprilSylph/XKit-Rewritten#587] [Suggestion Box]
Could we get a tweak to hide this panel?
![image](https://user-images.githubusercontent.com/68124115/167514731-3d91f77c-364a-45d5-994d-1168d82aa98b.png)

(It's funny to me that I want to hide this, because I used to like having tracked tags there from the earlier version of Tag Tracking+... but this isn't the whole list and Tumblr's idea of "recent" posts seems to be different than "ones that are actually new to me since the last time I loaded the tag page", so to me it's not useful and is just visual clutter.)

**Comment by marcustyphoon** [comment]
Do send your feedback to Staff about what would need to be done to it so that you'd like keeping it there! As far as I know, this is still an experiment. It sounds like a count of posts since you last viewed that tag is, at the very least, not impossible.

(But, +1 from me!)

**Comment by arcanetrivia** [comment]
Seems like it was just a test and they've pulled it already (https://changes.tumblr.com/post/684158607659614208/friday-may-13th-2022), but I did send them feedback about what would have made it useful. If it acted more like Tag Tracking+ (showed all my followed tags in alphabetical order, with a count of posts I haven't seen) that would be a great native feature.
